### PR TITLE
Skip the install of a few big conda env

### PR DESCRIPTION
**Pull request by abretaud**
Skip the install of a few big conda env to reduce the image size. Currently the image does not build anymore on quay.io because it's too big

### CODE REVIEW

**Review by hexylena — APPROVED** [review]

**Comment by bgruening** [comment]
The assumption is that auto_install will do the trick for us? Like it!

**Comment by abretaud** [comment]
Yep that's it, auto_install is on in this flavor

File: genome_annotation_tools_3.yml [hunk 1]
```diff
@@ -94,7 +97,7 @@ tools:
   tool_panel_section_label: "Assembly"
 
 - name: cdhit
-  owner: jjohnson
+  owner: bebatut
```
Comment (bgruening) [inline genome_annotation_tools_3.yml:100]: I think this should be IUC https://github.com/galaxyproject/tools-iuc/pulls?q=is%3Apr+cdhit+is%3Aclosed

Comment (abretaud) [reply genome_annotation_tools_3.yml:100]: ah yes sorry, I missed this one, I will correct this

**Pull request merged**